### ISSUE TITLE
Questions about loss

### ISSUE THREAD

**Issue by yanghaibin-cool** [ifzhang/ByteTrack#70]
cd <ByteTrack_HOME>
python3 tools/train.py -f exps/example/mot/yolox_s_mix_det.py -d 1 -b 4 --fp16 -o -c pretrained/yolox_s.pth
then i get the following training results:
![image](https://user-images.githubusercontent.com/72022193/141397752-5c32c164-8633-4ca7-b0aa-40e8fb8cd6b1.png)
![image](https://user-images.githubusercontent.com/72022193/141397779-d33c11ac-2cb1-4c8b-bd1f-88944c7c5e8b.png)
I feel the training loss is problematic,this training loss has been oscillating and l1 loss=0,What is the reason for this？

**Comment by ifzhang** [comment]
We only use l1 loss in the last 10 epochs. It is normal that it is 0 in the first 70 epochs.

**Comment by yanghaibin-cool** [comment]
> 

Thank you. I understand now.
